tools/java_api/Makefile

```diff
@@ -0,0 +1,206 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Since this `Makefile` is generated by CMake, it should be ignored and deleted from git as it can be automatically recreated.